docs/buildingcrawler.md

```diff
@@ -371,8 +371,13 @@ parse:
       author: .//meta[@name="author"]/@content
       publishedAt: .//*[@class="date"]/text()
       description: .//meta[@property="og:description"]/@content
+    keys:
```

Comment:
I would suggest we use syntax similar to ftm mappings for consistency. Something like https://github.com/alephdata/aleph/blob/main/mappings/md_companies.yml#L15-L17

So the keys section will look like:
```
keys:
  - title
  - author
```

Reply:
The `keys` section needs to be updated now I think?